website/docs/gas.md

```diff
@@ -0,0 +1,33 @@
+# Gas fees update
+
+The gas fees update endpoint allows users to modify the current gas prices on a running devnet. This feature is particularly useful for testing purposes and for adjustments needed after forking to align with the forked network's gas prices.
+
+JSON-RPC Request
+The following JSON-RPC request can be used to update gas prices:
```

Comment:
Try rendering this (e.g. with ctrl+shift+v in vs-code, or by clicking View File in GitHub). It doesn't look good. I'd suggest using `## Request`. Same for Respone in L23.

Reply:
Fixed in 0902548cf65b92163ef4944d0827273b7708052d